bin/gem2rpm

```diff
@@ -45,6 +60,11 @@ end
 srpmdir = nil
 specfile = nil
 if options[:srpm]
+  unless File.exist?(gemfile)
```

Comment:
1. => Yes, I agree with you. I will move its logic.
2. => Yes, I will remove it from this PR, and open new separate PR, and I will consider other hundled exceptions.